packages/integration-tests/src/test-project-setup/data_storage_auth_with_triggers.ts

```diff
@@ -642,4 +653,84 @@ class DataStorageAuthWithTriggerTestProject extends TestProjectBase {
       );
     }
   };
+
+  private assertCustomEmailSenderWorks = async (
+    backendId: BackendIdentifier
+  ) => {
+    const TIMEOUT_MS = 1000 * 60 * 2; // 2 minutes
+    const startTime = Date.now();
+    const queue = await this.resourceFinder.findByBackendIdentifier(
+      backendId,
+      'AWS::SQS::Queue',
+      (name) => name.includes('testFuncQueue')
+    );
```

Comment:
We should add separate queue for this scenario.

Otherwise messages from email and scheduled functions will land in the same queue and we can't tell which one is which.

Reply:
It seems that sequencing of post deployment assertions help. But it's very implicit assumption. Using separate resources for both scenarios will remove that trap.